CHANGELOG

```diff
@@ -1,6 +1,9 @@
 devel
 -----
 
+* Retain backwards-compatibility for client tools when more than one
```

Comment:
I would join this changelog with the original changelog entry, since they have never been visible to end-users.